sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchOptions.java

```diff
@@ -192,26 +194,26 @@
     private Integer top;
 
     /**
-     * Get the includeTotalResultCount property: A value that specifies whether to fetch the total count of results.
-     * Default is false. Setting this value to true may have a performance impact. Note that the count returned is an
+     * Get the includeTotalCount property: A value that specifies whether to fetch the total count of results. Default
+     * is false. Setting this value to true may have a performance impact. Note that the count returned is an
      * approximation.
      *
-     * @return the includeTotalResultCount value.
+     * @return the includeTotalCount value.
      */
-    public Boolean isIncludeTotalResultCount() {
-        return this.includeTotalResultCount;
+    public Boolean isTotalCountIncluded() {
+        return this.includeTotalCount;
     }
 
     /**
-     * Set the includeTotalResultCount property: A value that specifies whether to fetch the total count of results.
-     * Default is false. Setting this value to true may have a performance impact. Note that the count returned is an
+     * Set the includeTotalCount property: A value that specifies whether to fetch the total count of results. Default
+     * is false. Setting this value to true may have a performance impact. Note that the count returned is an
      * approximation.
      *
-     * @param includeTotalResultCount the includeTotalResultCount value to set.
+     * @param includeTotalCount the includeTotalCount value to set.
      * @return the SearchOptions object itself.
      */
-    public SearchOptions setIncludeTotalResultCount(Boolean includeTotalResultCount) {
-        this.includeTotalResultCount = includeTotalResultCount;
+    public SearchOptions setIncludeTotalCount(Boolean includeTotalCount) {
```

Comment:
the names of getter and setter don't match. Can this be updated to `setTotalCountIncluded`?

Reply:
The reason for the mismatch is I'm trying to get this more inline with the hand-written class to hopefully replace that soon with this generated class